index.js

```diff
@@ -4,9 +4,11 @@ var path = require('path');
 module.exports = {
   name: 'ember-waypoints',
 
-  contentFor: function(type) {
-    if (type === 'vendor-suffix') {
-      return 'if (typeof(document) !== "undefined") {\n' + fs.readFileSync(path.join(__dirname, 'vendor/jquery-waypoints/waypoints.js')) + '\n}';
+  included: function(app) {
+    this._super.included(app);
+
+    if (!process.env.EMBER_CLI_FASTBOOT) {
+      app.import(app.bowerDirectory + '/jquery-waypoints/waypoints.min.js');
```

Comment:
@rwjblue like this right? contentFor should not be necessary since I'm not using vendor--I'm using waypoints from bower (I have a blueprint for loading this from bower)